open_academy/views/open_academy_menus_views.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <menuitem id="open_academy_menu_root" name="Open Academy" sequence="1"/>
-    <menuitem id="open_academy_course_menu" name="Courses" parent="open_academy_menu_root" action="open_academy_course_action" sequence="1"/>
-    <menuitem id="open_academy_session_menu" name="Sessions" parent="open_academy_menu_root" action="open_academy_session_action" sequence="2"/>
+    <menuitem id="open_academy_board_session_menu" name="Sessions Dashboard" parent="open_academy_menu_root" action="open_academy_board_session_action" sequence="1"/>
```

Comment:
You don't need to move this menu, actually this need to be the first because is the parent of all the menus. Move the sequence value.

```suggestion
    <menuitem id="open_academy_menu_root" name="Open Academy" sequence="1"/>
    <menuitem id="open_academy_board_session_menu" name="Sessions Dashboard" parent="open_academy_menu_root" action="open_academy_board_session_action" sequence="1"/>
    <menuitem id="open_academy_course_menu" name="Courses" parent="open_academy_menu_root" action="open_academy_course_action" sequence="2"/>
    <menuitem id="open_academy_session_menu" name="Sessions" parent="open_academy_menu_root" action="open_academy_session_action" sequence="3"/>
```